es6-lib/decoders/ldjson.js

```diff
@@ -0,0 +1,57 @@
+import _ from 'underscore';
+import {
+  Transform
+}
+from 'stream';
+import config from '../config';
+
+class LDJson extends Transform {
+
+  constructor(columns) {
+    super({
+      writableObjectMode: false,
+      readableObjectMode: true,
+      highWaterMark: config().rowBufferSize
+    });
+    this._buf = '';
+  }
+
+  _push(token) {
+    try {
+      this.push(JSON.parse(token));
+    } catch(e) {
+
```

Comment:
this is a `''` token

i suppose there could be a warning if it fails on something other than `''`